### PR TITLE
Make naming consistent within contact fragment

### LINKED ISSUE #243: Make naming consistent within contact fragment
**Is this a BUG REPORT or FEATURE REQUEST?**:
 feature

**What happened**:
Buttons should have consistent naming i.e "text"
**What you expected to happen**:
Change button key to "text"

### PR DESCRIPTION
**Pull request by lauragift21**
**What this PR does / why we need it**:
It fixes the naming convention used within contact fragment

**Special notes for your reviewer**:

**Release note**:

```release-note
Make naming consistent within contact fragment
```

### CODE REVIEW

File: exampleSite/content/_index/contact.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ subtitle  = "*not working on demo page*"
 # PostURL can be used with backends such as mailout from caddy
 post_url = "https://example.com/mailout" #default: formspree.io
 email = "mail@example.com"
-button = "Send Button" # defaults to theme default
+text = "Send Button" # defaults to theme default
```
[the remark this answers is not in the document]
Comment (stp-ip) [reply exampleSite/content/_index/contact.md]: @mpourismaiel 
```
[buttons]
  text = ""
```
or
```
button_text = ""
```
or keep
```
button = ""
```

[the remark this answers is not in the document]
Comment (mpourismaiel) [reply exampleSite/content/_index/contact.md]: I like `button_text` a lot more. It's easier to write, specially if the user is writing JSON for frontmatter.

[the remark this answers is not in the document]
Comment (stp-ip) [reply exampleSite/content/_index/contact.md]: I might agree. It uses the _text part to be closer to the other buttons, but makes it flat and easier in this case. @lauragift21 would you agree?

[the remark this answers is not in the document]
Comment (lauragift21) [reply exampleSite/content/_index/contact.md]: Yeah, I agree. I'll merge in the current change.

**Pull request merged**